podcasts/End of Year/Stories/YearOverYearStory.swift

```diff
@@ -0,0 +1,211 @@
+import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
+
+struct YearOverYearStory: ShareableStory {
+    var duration: TimeInterval = 5.seconds
+
+    let identifier: String = "year_over_year"
+
+    let yearOverYearListeningTime: YearOverYearListeningTime
+
+    let subscriptionTier: SubscriptionTier = SubscriptionHelper.subscriptionTier
+
+    var title: String {
+        let listeningPercentage = yearOverYearListeningTime.percentage
+        switch listeningPercentage {
+        case _ where listeningPercentage == .infinity:
+            return L10n.eoyYearOverYearTitleSkyrocketed
+        case _ where listeningPercentage > 10:
+            return L10n.eoyYearOverYearTitleWentUp("\(listeningPercentage.clean)%")
+        case _ where listeningPercentage < 0:
+            return L10n.eoyYearOverYearTitleWentDown
+        default:
+            return L10n.eoyYearOverYearTitleFlat
+        }
+    }
+
+    var subtitle: String {
+        let listeningPercentage = yearOverYearListeningTime.percentage
+        switch listeningPercentage {
+        case _ where listeningPercentage > 10:
+            return L10n.eoyYearOverYearSubtitleWentUp
+        case _ where listeningPercentage < 0:
+            return L10n.eoyYearOverYearSubtitleWentDown
+        default:
+            return L10n.eoyYearOverYearSubtitleFlat
+        }
```

Comment:
We can achieve the same result a bit cleaner without the `where`:
 
```suggestion
        switch yearOverYearListeningTime.percentage {
        case 10...:
            L10n.eoyYearOverYearSubtitleWentUp
        case ...0:
            L10n.eoyYearOverYearSubtitleWentDown
        default:
            L10n.eoyYearOverYearSubtitleFlat
        }
```